Index: generic/proof-site.el
===================================================================
RCS file: /home/proofgen/src/ProofGeneral/generic/proof-site.el,v
retrieving revision 4.26
diff -c -r4.26 proof-site.el
*** generic/proof-site.el	2000/05/25 14:52:30	4.26
--- generic/proof-site.el	2000/05/25 20:14:27
***************
*** 99,105 ****
    :group 'proof-general-internals)
  
  (defcustom proof-info-directory
!   (concat proof-home-directory "doc/")
    "Where Proof General Info files are installed. Ends with slash."
    :type 'directory
    :group 'proof-general-internals)
--- 99,105 ----
    :group 'proof-general-internals)
  
  (defcustom proof-info-directory
!   "/usr/info/"
    "Where Proof General Info files are installed. Ends with slash."
    :type 'directory
    :group 'proof-general-internals)
