diff -cr ProofGeneral-2.0/coq/coqtags ProofGeneral-2.0.new/coq/coqtags
*** ProofGeneral-2.0/coq/coqtags	Thu Sep 24 16:36:20 1998
--- ProofGeneral-2.0.new/coq/coqtags	Thu Sep 24 17:04:00 1998
***************
*** 1,4 ****
! #!/usr/local/bin/perl4
  #
  # $Id: coqtags,v 2.1 1998/09/09 14:02:29 da Exp $
  #
--- 1,4 ----
! #!/usr/bin/perl
  #
  # $Id: coqtags,v 2.1 1998/09/09 14:02:29 da Exp $
  #
diff -cr ProofGeneral-2.0/lego/legotags ProofGeneral-2.0.new/lego/legotags
*** ProofGeneral-2.0/lego/legotags	Thu Sep 24 16:36:30 1998
--- ProofGeneral-2.0.new/lego/legotags	Thu Sep 24 17:04:10 1998
***************
*** 1,4 ****
! #!/usr/local/bin/perl
  #
  # $Id: legotags,v 2.1 1998/09/09 14:02:46 da Exp $
  #
--- 1,4 ----
! #!/usr/bin/perl
  #
  # $Id: legotags,v 2.1 1998/09/09 14:02:46 da Exp $
  #

