diff -cr ProofGeneral.old/coq/coqtags ProofGeneral/coq/coqtags
*** ProofGeneral.old/coq/coqtags	Wed Oct  7 18:36:57 1998
--- ProofGeneral/coq/coqtags	Wed Oct  7 19:04:35 1998
***************
*** 1,4 ****
! #!/usr/local/bin/perl
  
  undef $/;
  
--- 1,4 ----
! #!/usr/bin/perl
  
  undef $/;
  
diff -cr ProofGeneral.old/generic/proof-site.el ProofGeneral/generic/proof-site.el
*** ProofGeneral.old/generic/proof-site.el	Wed Oct  7 18:38:21 1998
--- ProofGeneral/generic/proof-site.el	Wed Oct  7 19:05:08 1998
***************
*** 76,82 ****
    :group 'proof-internal)
  
  (defcustom proof-info-directory
!   (concat proof-home-directory "doc/")
    "*Where Proof General Info files are installed."
    :type 'directory
    :group 'proof-internal)
--- 76,82 ----
    :group 'proof-internal)
  
  (defcustom proof-info-directory
!   "/usr/info/"
    "*Where Proof General Info files are installed."
    :type 'directory
    :group 'proof-internal)
diff -cr ProofGeneral.old/lego/legotags ProofGeneral/lego/legotags
*** ProofGeneral.old/lego/legotags	Wed Oct  7 18:37:18 1998
--- ProofGeneral/lego/legotags	Wed Oct  7 19:04:35 1998
***************
*** 1,2 ****
! #!/usr/local/bin/perl
  #
--- 1,2 ----
! #!/usr/bin/perl
  #
